sdk/nodejs/tsconfig.json

```diff
@@ -99,10 +99,9 @@
         "tests/unwrap.spec.ts",
         "tests/util.ts",
         "tests/runtime/asyncIterableUtil.spec.ts",
-        "tests/runtime/closureLoader.spec.ts",
```

Comment:
Can this file be deleted?

Reply:
Weird. It is deleted, but Git flags it as rename, guess there's just enough similarity to trip it up
```
renamed: sdk/nodejs/tests/runtime/closureLoader.spec.ts ⟶   sdk/nodejs/tests/runtime/testdata/closure-tests/cases/152-Capture-config-created-on-the-inside/deploymentOnlyModule/index.ts
```